explorer/app/common/context/exploreState.tsx

```diff
@@ -163,31 +162,29 @@ export type ExploreState = {
   loading: boolean;
   paginationState: PaginationState;
   relatedListItems: RelatedListItems;
-  sortState: ColumnSort | undefined; // TODO update sortState to ColumnSort[]?
+  sorting: ColumnSort[];
```

Comment:
Updated `sorting` to always be defined - if the configuration "default" is undefined, we return an empty array. This is to support future use of `resetSorting` functions.